tests/lib/newlib/newlib_hook/testcase.yaml

```diff
@@ -0,0 +1,10 @@
+tests:
+  libraries.libc.newlib.userspace:
+    platform_exclude: mps2_an521_ns
+    filter: CONFIG_ARCH_HAS_USERSPACE and TOOLCHAIN_HAS_NEWLIB == 1
+    tags: clib newlib userspace
+  libraries.libc.newlib:
+    filter: TOOLCHAIN_HAS_NEWLIB == 1
+    tags: clib newlib
+    extra_configs:
+      - CONFIG_TEST_USERSPACE=n
```

Comment:
Remove `CONFIG_TEST_USERSPACE=y` from `prj.conf` and add `CONFIG_TEST_USERSPACE=y` and other userspace-specific configs (e.g. `CONFIG_HEAP_MEM_POOL_SIZE`) to `extra_configs` of `libraries.libc.newlib.userspace`.

Reply:
Thanks you for you comment. Testcase.yaml has been updated. Because I just test write stub, CONFIG_HEAP_MEM_POOL_SIZE is not needed.